### ISSUE TITLE
If props pass a new items array the wall falls with item not found

### ISSUE THREAD

**Issue by moetun88** [fuxingloh/vue-masonry-wall#16]
If a parent element passes in an entirely new array (props) the masonry wall does not redraw

**Comment by moetun88** [comment]
In an implementation which isn't auto loading via append simply watching for items and redrawing is a workaround. 
Perhaps an additional flag to indicate between the two types of updates items can experience?

**Comment by forrust** [comment]
Yes i'm experiencing the same issue, as soon as my (store) items array changes the masonry wall collapse in to an error. Does looks very promising tho!
